SA/design_concept/content/design/design_evaluation.typ

```diff
@@ -1,19 +1,20 @@
 = Design Evaluation Criteria
+This section describes the decision process used to decide on a final design for function & type composition in VisualFP.
 
-The following code scenarios were defined to evaluate visualization designs
-regarding their simplicity and clarity of the underlying functional
-concepts. The decision for which design is worth pursuing further will be
-made based on how well a design is able to represent these scenarios.
+Unfortunately, it isn't possible to compare 'how good' a design is in an objective fashion.
+This is why the decision process is based on selected code scenarios and a questionnaire to evaluate their visual representation.
```

Comment:
```suggestion
This is why the decision process is based on selected code scenarios and a non-quantitative questionnaire to evaluate their visual representation.
```

A questionnaire is usually quantitative, this one is not. I would somehow work this into this text